static/js/src/advantage/offers/types.ts

```diff
@@ -15,4 +15,5 @@ export type Offer = {
   items: Item[];
   marketplace: UserSubscriptionMarketplace;
   total: number;
+  discount?: number | null;
```

Comment:
The key "discount" will always exist? Should this be `discount` without `?`
```suggestion
  discount: number | null;
```